rerun_cpp/src/rerun/datatypes/image_format_ext.cpp

```diff
@@ -41,6 +41,24 @@ namespace rerun {
                   color_model(color_model_),
                   channel_datatype(datatype_) {}
 
+            /// How many bytes will this image occupy?
+            size_t num_bytes() const {
+                return width * height * this->bits_per_pixel() / 8;
+            }
+
+            /// How many bits per pixel?
+            ///
+            /// Note that this is not necessarily a factor of 8.
+            size_t bits_per_pixel() const {
```

Comment:
Urgh, I dislike the notion of bits per pixel.
We had the same issue on wgpu where contributor wanted to have bytes per pixel but then block compression is a thing and the whole notion of pixel gets super messy - as is already with chroma downsampling! Most block compression formats still have non-fractional bits per decoded pixel, but at latest when ASTC's variable block size enters the picture everything breaks down.
I'd rather we not have this and instead have a method on pixel format that gives you the expected size for under a resolution + format. The model+datatype path is ofc trivial and can stay here.

Reply:
fixed!

Reply:
<3